### ISSUE TITLE
Issues with running v1.2.0 on Sentinnel 2 data

### ISSUE THREAD

**Issue by krishanr** [NRCan/geo-deep-learning#209]
Hello everyone,

I've been attempting to run v1.2.0 of the geo-deep-learning code on Sentinnel 2 data downloaded using the FireHR package. There are two issues I encountered, one of which I was able to bypass:

1. The image data are float32 type between 0 and 1.
   So I had to remove the lines:
```
assert np_array.dtype in ['uint8', 'uint16'], f"Invalid datatype {np_array.dtype}. " \
													  f"Only uint8 and uint16 are supported in current version"
```
from utils/readers.py, and lines 443-445 in utils/utils.py, to deal with float data.
2. I'm able to get images_to_samples.py to work, but train_segmentation.py doesn't work, largely because the val_loss is over e^20 on every epoch (so the models never get saved).

A side note is that I was able to train a model using code from around January 2020.

Any ideas?

**Comment by victorlazio109** [comment]
Hi Krishan,

On first look, it is possible the images which are already scaled between [0, 1] are being re-scaled again before it's fed into the model. Take a look at augmentation.py script (GDL latest commits/version) there is a scaling function there which uses a scale parameter set from a config file; it may be worthwhile to comment the function out.

Also, just to eliminate problems with data type you could debug by scaling back to uint8 (x 255) before your start the GDL pipeline; test to see if the val loss is still exploding.

**Comment by krishanr** [comment]
Ok thanks Mathieu, I'll try treating it as unit16. Any parameters I should set in the yaml file for this? I'm guessing `scale_data: [0,1]` is sufficient?
